apps/pano/src/routes/send.tsx

```diff
@@ -12,15 +6,21 @@ import {
   Input,
   Label,
   ValidationMessage,
-} from "~/ui-library";
+} from "@kampus/ui";
+import type { ActionFunction } from "@remix-run/node";
+import { json, redirect } from "@remix-run/node";
+import { useActionData, useFetcher, useTransition } from "@remix-run/react";
+import normalizeUrl from "normalize-url";
+import { createPost } from "~/models/post.server";
+import { requireUserId } from "~/session.server";
 import { validateURL } from "~/utils";
 
 export const action: ActionFunction = async ({ request }) => {
   const formData = await request.formData();
   const title = formData.get("title");
   const url = formData.get("url");
 
-  if (!validateURL(url)) {
+  if (!validateURL(url?.toString() ?? "")) {
```

Comment:
I think we could handle `null` use case inside of this function, if we need to call this function more than once with `nullable` case, code can be verbose

Reply:
yes, this will be addressed when content for posts is enabled with my PR. Good catch! 💯